spec/commands/newgem_spec.rb

```diff
@@ -116,19 +119,20 @@ def gem_skeleton_assertions(gem_name)
     let(:gem_name) { "test_gem" }
     let(:generated_gem) { Bundler::GemHelper.new(bundled_app(gem_name).to_s) }
 
-    context "git config user.name present" do
+    context "git config github.user present" do
       before do
         execute_bundle_gem(gem_name)
       end
 
       it "contribute URL set to git username" do
         expect(bundled_app("test_gem/README.md").read).not_to include("[USERNAME]")
+        expect(bundled_app("test_gem/README.md").read).to include("github.com/bundleuser")
       end
     end
 
-    context "git config user.name is absent" do
+    context "git config gihhub.user is absent" do
       before do
-        `git config --unset user.name`
+        `git config --unset github.user`
```

Comment:
This exact syntax with `git` is used 5 times in this file.  Is there a new policy written somewhere for this change to be enacted?